presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -999,39 +999,71 @@ public void testCountColumn()
     }
 
     @Test
-    public void testWildcard()
+    public void testSelectAllFromTable()
     {
         assertQuery("SELECT * FROM orders");
-    }
 
-    @Test
-    public void testMultipleWildcards()
-    {
+        // multiple wildcards
         assertQuery("SELECT *, 123, * FROM orders");
-    }
 
-    @Test
-    public void testMixedWildcards()
-    {
+        // mixed wildcards
         assertQuery("SELECT *, orders.*, orderkey FROM orders");
-    }
 
-    @Test
-    public void testQualifiedWildcardFromAlias()
-    {
+        // qualified wildcard from alias
         assertQuery("SELECT T.* FROM orders T");
-    }
 
-    @Test
-    public void testQualifiedWildcardFromInlineView()
-    {
+        // qualified name from alias, column aliases
+        assertQuery("SELECT T.* FROM (VALUES (1, 2, 3)) T (a, b, c)", "SELECT 1 a, 2 b, 3 c");
+
+        //qualified wildcard from inline view
         assertQuery("SELECT T.* FROM (SELECT orderkey + custkey FROM orders) T");
+
+        // qualified wildcard
+        assertQuery("SELECT orders.* FROM orders");
     }
 
     @Test
-    public void testQualifiedWildcard()
+    public void testSelectAllFromRow()
     {
-        assertQuery("SELECT orders.* FROM orders");
+        // named row fields and aliases
+        assertQuery("SELECT CAST((ROW (1, 'a')) AS ROW (f1 bigint, f2 varchar(1))).* AS (alias1, alias2)", "SELECT 1 alias1, 'a' alias2");
```

Comment:
Move these tests to something under `io.prestosql.sql.query` so that they don't run with every connector (given that they don't need to access tables)